src/physics.F90

```diff
@@ -390,6 +399,20 @@ subroutine scatter(p, i_nuclide)
     ! Set event component
     p % event = EVENT_SCATTER
 
+    ! sample new outgoing angle for isotropic in lab scattering
+    if (materials(p % material) % p0(i_nuc_mat)) then
+
+      ! sample isotropic-in-lab outgoing direction
+      uvw_new(1) = TWO * prn() - ONE
+      phi = TWO * PI * prn()
+      uvw_new(2) = cos(phi) * sqrt(ONE - uvw_new(1)*uvw_new(1))
+      uvw_new(3) = sin(phi) * sqrt(ONE - uvw_new(1)*uvw_new(1))
+      mu_lab = dot_product(uvw_old, uvw_new)
+
+      ! change direction of particle
+      p % coord(1) % uvw = rotate_angle(uvw_new, mu_lab)
```

Comment:
@walshjon - I *think* that my latest commit revises this code block as you've advised.